test/plots/date-intervals.ts

```diff
@@ -0,0 +1,28 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export async function intervalWeekMonth() {
```

Comment:
Generally speaking it’s nice if the test name starts with the file name, so that the tests are easier to find. For example, this could be called `dateIntervalWeekMonth` or some such.